src/app/domains/categories/domains.transaction-categories.gateway.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { fakeRequest } from 'src/app/common/http/common.http.fake-request';
+import { transactionCategoriesMock } from './domains.transaction-categories.mocks';
+import { ITransactionCategoryApiResponse } from './domains.transaction-categories.types';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DomainsTransactionCategoriesGateway {
+  public constructor(
+    private readonly http: HttpClient,
+  ) { }
+
+  public getTransactionCategories(): Observable<ITransactionCategoryApiResponse[]> {
+    return fakeRequest(transactionCategoriesMock);
+    //return this.http.get<ITransactionCategoryApiResponse[]>(API_TRANSACTION_CATEGORIES_URL);
```

Comment:
We will actually leave it commented until BE is implemented. So you don't have to later uncomment this before merging. Please only adjust tests for it in a way that we can also later just uncomment something and it will pass both now and then.